auth-server/src/main/java/com/fiftyfive/authserver/controllers/AuthController.java

```diff
@@ -0,0 +1,39 @@
+package com.fiftyfive.authserver.controllers;
+
+import com.fiftyfive.authserver.dtos.*;
+import com.fiftyfive.authserver.services.AuthService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/auth")
+public class AuthController {
+    @Autowired
+    private AuthService authService;
```

Comment:
we can use constructor injection here, instead of using @Autowired annotation

Reply:
yes,we can.will make change

Reply:
if I am removing @Autowired and putting the different args constructor for testing purpose ,its not working